modules/schema/src/category/gis.ts

```diff
@@ -8,6 +9,62 @@ export type {GeoJSON, Feature, Geometry, Position, GeoJsonProperties} from 'geoj
 // eslint-disable-next-line import/no-unresolved
 export type {Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon} from 'geojson';
 
+// Aggregate information for converting GeoJSON into other formats
+export type GeojsonGeometryInfo = {
+  coordLength: number;
+  pointPositionsCount: number;
+  pointFeaturesCount: number;
+  linePositionsCount: number;
+  linePathsCount: number;
+  lineFeaturesCount: number;
+  polygonPositionsCount: number;
+  polygonObjectsCount: number;
+  polygonRingsCount: number;
+  polygonFeaturesCount: number;
+};
+
+// FLAT GEOJSON FORMAT GEOMETRY
+export type FlatGeometryType = 'Point' | 'LineString' | 'Polygon';
+type RemoveCoordinatesField<Type> = {
+  [Property in keyof Type as Exclude<Property, 'coordinates'>]: Type[Property];
+};
+
+/**
+ * GeoJSON (Multi)Point geometry with coordinate data flattened into `data` array and indexed by `indices`
+ */
+export type FlatPoint = RemoveCoordinatesField<Point> & {
+  data: number[];
```

Comment:
I feel a type like this is used in several places. You've removed it and it'es repeated in several places.

```typescript
type geomBinary = {
     data: number[];
     indices: number[];
}
```

Reply:
Added back in